data/railsconf/railsconf-2023/videos.yml

```
@@ -0,0 +1,1044 @@
+    1,400-controller Ruby on Rails monolith accessible.
+  thumbnail_xs: https://i.ytimg.com/vi/4j2zlvE_Yj8/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/4j2zlvE_Yj8/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/4j2zlvE_Yj8/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/4j2zlvE_Yj8/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/4j2zlvE_Yj8/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 4j2zlvE_Yj8
+- title: ActiveRecord::Encryption; Stop Hackers from...
+  raw_title: RailsConf 2023 - ActiveRecord::Encryption; Stop Hackers from... by Matthew
+    Langlois, Kylie Stradley
+  speakers:
+    - Matthew Langlois
+    - Kylie Stradley
+  description: |-
+    ActiveRecord::Encryption; Stop Hackers from Reading your Data by Matthew Langlois, Kylie Stradley
+
+    Have you ever wondered how to encrypt data in your Rails application but weren’t sure where to get started? We’ll briefly talk about why you would want to encrypt data, and then discuss how you can get started with encrypting columns in your Rails application including pitfalls and successes we encountered while implementing ActiveRecord::Encryption at GitHub. Attendees will be confident in making a decision to implement ActiveRecord::Encryption in their application.
+  thumbnail_xs: https://i.ytimg.com/vi/4mZNP_Dgi2w/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/4mZNP_Dgi2w/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/4mZNP_Dgi2w/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/4mZNP_Dgi2w/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/4mZNP_Dgi2w/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 4mZNP_Dgi2w
+- title: Applying microservices patterns to a modular monolith
+  raw_title: RailsConf 2023 - Applying microservices patterns to a modular monolith
+    by Guillermo Aguirre
+  speakers:
+    - Guillermo Aguirre
+  description: |-
+    Dealing with legacy monoliths can be challenging due to accumulated technical debt. When faced with this situation, we usually start looking at different strategies to solve this, like microservices, engines, and modularization, just to name a few.
+
+    When choosing to follow a modular approach, I explored applying microservices patterns to our modular monolith. In this talk, I’ll share my experience in adopting this approach, focusing on ensuring data consistency and distributed transactions, as well as the pitfalls and insights I discovered along the way.
+  thumbnail_xs: https://i.ytimg.com/vi/4zrQAJ0RlI4/default.jpg
+  thumbnail_sm: https://i.ytimg.com/vi/4zrQAJ0RlI4/mqdefault.jpg
+  thumbnail_md: https://i.ytimg.com/vi/4zrQAJ0RlI4/hqdefault.jpg
+  thumbnail_lg: https://i.ytimg.com/vi/4zrQAJ0RlI4/sddefault.jpg
+  thumbnail_xl: https://i.ytimg.com/vi/4zrQAJ0RlI4/maxresdefault.jpg
+  published_at: '2023-07-10'
+  video_id: 4zrQAJ0RlI4
+- title: 'Bridging the Gap: Creating Trust Between Non-Technical...'
```

It also looks like all titles ending in `...` are cut off in the YouTube title itself
```suggestion
- title: Bridging the Gap: Creating Trust Between Non-Technical Stakeholders and Engineering Teams
```